### ISSUE TITLE
Signing error: bundle format unrecognized, invalid, or unsuitable

### ISSUE THREAD

**Issue by pablog** [rollbar/rollbar-ios#21]
This is happening to me when trying to archive my application in order to upload it to AppStore (adding the .framework manually):

```
CodeSign PROJECT_PATH/Frameworks/Rollbar.framework
    cd PROJECT_PATH
    export CODESIGN_ALLOCATE=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/codesign_allocate
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"

Signing Identity:     "iPhone Distribution: MY_SIGNING_IDENTITY"

/usr/bin/codesign --force --sign 61430F105530A00798A5E6975308AFF65A9AC0A9 --preserve-metadata=identifier,entitlements --timestamp=none DERIVED_DATA_PROJECT_PATH/Frameworks/Rollbar.framework

DERIVED_DATA_PROJECT_PATH/Frameworks/Rollbar.framework: bundle format unrecognized, invalid, or unsuitable
Command /usr/bin/codesign failed with exit code 1
```
Instead, when adding the project using cocoa pods, I have the BITCODE issue. I did what the documentation says (disabling bitcode for my target, which I did), but problem is still there.

```
ld: 'PROJECT_PATH/Pods/Rollbar/Vendor/CrashReporter.framework/CrashReporter(libCrashReporter-iphoneos.a-armv7-master.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I really appreaciate any help.

**Comment by brianr** [comment]
@pablog Apologies for our very slow response here. Is this still an issue for you?

**Comment by pablog** [comment]
Yes, it is. It should be fixed after merging https://github.com/rollbar/rollbar-ios/pull/26

**Comment by ezarowny** [comment]
Can you try using the version that was just merged to master?

**Comment by ezarowny** [comment]
FYI, I'm working to recompile PLCrashReporter so we can use a different prefix than `BIT`.

**Comment by pablog** [comment]
I'm trying the version in master, but it fails with the error:

```
Dependency "rollbar-ios" has no shared framework schemes for any of the platforms: iOS

If you believe this to be an error, please file an issue with the maintainers at https://github.com/rollbar/rollbar-ios/issues/new
```

Is this fixed in #25?

**Comment by pablog** [comment]
This is working great now! Can close this issue.

It'd be good to make a new release at this point.

**Comment by ezarowny** [comment]
@pablog sorry for the delay. I was on vacation 🌴 

Are you using Release or Archive when you build the Rollbar lib?

**Comment by pablog** [comment]
@ezarowny I'm using Carthage to include the framework (branch master).

**Comment by aavina** [comment]
I'm also receiving the linker issue when integrating Rollbar with an iOS9 Swift 2.x project. I'm using Cocoapods. I can't build at all right from the start after including Rollbar in my Podfile, pod installing, and building.

**Comment by ezarowny** [comment]
@pablog do you know how Carthage is building the library? I'm guessing it's Archive but I haven't read any of the Carthage source and I've never used it.

**Comment by ezarowny** [comment]
Can you try with the version in #29 ?

**Comment by ezarowny** [comment]
Should be fixed with #29 and 0.2.0
